src/components/Button/Button.jsx

```diff
@@ -114,7 +114,7 @@ const Button = createClass({
 
 		return (
 			<button
-				{...passThroughs}
+				{...omitProps(passThroughs, Button, ['callbackId'])}
```

Comment:
this pattern is a little problematic, sort of couples `Button` to `ButtonGroup`. we should try to think of another way to pass arbitrary data to callbacks